corehq/apps/domain/forms.py

```diff
@@ -954,12 +956,9 @@ def save(self, domain):
 pwd_pattern = re.compile( r"([-\w]){"  + str(min_pwd) + ',' + str(max_pwd) + '}' )
```

Comment:
This doesn't seem to be used anymore and can be deleted (along with the 2 lines above it)